src/main/java/jp/ikedam/jenkins/plugins/updatesitesmanager/internal/IgnoreNotPOST.java

```diff
@@ -0,0 +1,39 @@
+package jp.ikedam.jenkins.plugins.updatesitesmanager.internal;
+
+import hudson.util.HttpResponses;
+import jp.ikedam.jenkins.plugins.updatesitesmanager.UpdateSitesManager;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.interceptor.Interceptor;
+import org.kohsuke.stapler.interceptor.InterceptorAnnotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.lang.reflect.InvocationTargetException;
+
+/**
+ * Just redirects to url of sites manager if not POST
+ *
+ * @author lanwen (Merkushev Kirill)
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+@Documented
+@InterceptorAnnotation(IgnoreNotPOST.Processor.class)
+public @interface IgnoreNotPOST {
```

Comment:
(replying to myself)
`@RequirePOST` displays an error page with a button to have users POST without any parameters, which should result users lose all managed update sites.
It is completely useless and we need another implementation (like this `@IgnoreNotPost`).